.gitignore

```diff
@@ -15,4 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-.idea
```

Comment:
It's fine to keep it...init removes it automatically. I use Webstorm, but I have my global .gitignore set to ignore .idea on all projects, so that I don't have to keep adding .idea to the .gitignore of every project I contribute to.  Just a thought. :)